src/main/java/demon/Commands.java

```diff
@@ -0,0 +1,285 @@
+        Deadline itemDeadline = new Deadline(description, dateTime);
+        tasks.getTaskList().add(itemDeadline);
+        String stringToSave = "D | " + (itemDeadline.getStatusIcon().equals("X") ? "1" : "0")
+                            + " | " + description + " | " + by + "\n";
+
+        // May produce IOException
+        storage.writeToFile(stringToSave);
+        Ui.printDivider();
+        Ui.showTaskAdded();
+        System.out.println(itemDeadline);
+        System.out.println("Now you have " + tasks.getTaskList().size() + " tasks in the list.");
+        Ui.printDivider();
+        Ui.promptNext();
+    }
+
+    /**
+     * Creates a To-do object as a task and stores into the taskList array.
+     * Method takes in the tasks arraylist and a string.
+     *
+     * @param tasks an array containing all tasks.
+     * @param input String input by the user containing the description.
+     * @throws EmptyDescriptionException if description not given.
+     * @throws IOException if fails to write new task into file.
+     */
+    public void addToDo(TaskList tasks, String input) throws EmptyDescriptionException, IOException {
+        String[] parts = input.split(" ",2);
+        if (parts.length < 2) {
+            throw new EmptyDescriptionException("Description is EMPTY!");
+        }
+        String toDo = parts[1];
+        Todo item_toDo = new Todo(toDo);
+        tasks.getTaskList().add(item_toDo);
+        String stringToSave = "T | " + (item_toDo.getStatusIcon().equals("X") ? "1" : "0") + " | " + toDo +"\n";
+        // May produce IOException
+        storage.writeToFile(stringToSave);
+        Ui.printDivider();
+        Ui.showTaskAdded();
+        System.out.println(item_toDo);
+        System.out.println("Now you have " + tasks.getTaskList().size() + " tasks in the list.");
+        Ui.printDivider();
+    }
+
+    /**
+     * Creates an Event object as a task and stores into the taskList array.
+     * Method takes in the tasks arraylist and a string.
+     *
+     * @param tasks An array containing all tasks.
+     * @param input String input by the user containing the description, date and time from, date and time to.
+     * @throws EmptyDescriptionException if description not given.
+     * @throws NoTimingException if date and time not given.
+     * @throws IOException if fails to write new task into file.
+     * @throws DateTimeParseException if date and time given is invalid.
+     */
+    public void addEvent(TaskList tasks, String input)
+            throws EmptyDescriptionException,
+                   NoTimingException,
+                   IOException,
+                   DateTimeParseException {
+        String[] parts = input.split(" ",2);
+        if (parts.length < 2) {
+            throw new EmptyDescriptionException("Description is EMPTY!");
+        }
+        String description_date = parts[1];
+        String[] parts2 = description_date.split("/from ");
```

Comment:
I feel this might be deviating from the coding style as well since `parts2` is not strictly in `camelCase`